design/schedule-unpaused-triggers-config_design.md

```diff
@@ -0,0 +1,102 @@
+# Schedule Unpaused Triggers Config Design
+## Abstract
+User want to configure what happens when schedule is unpaused. Either run backup immediately or at the next cron schedule.  This design proposes to add a new field to the schedule spec to allow user to configure the behavior
+
+## Background
+Currently, the default behavior is to run the trigger immediately and may not be desired by all users (https://github.com/vmware-tanzu/velero/issues/6517)
+
+## Goals
+- Make schedule unpaused behavior configurable
+
+## Non Goals
+- Changing the default behavior
+
+
+## High-Level Design
+Add a new field with to the schedule spec to allow user to configure the behavior defaulting to prior behavior of triggering backup immediately.
+
+## Detailed Design
+### CLI Changes
+`velero schedule unpause` will now take an optional bool flag `--unpause-triggers` to allow user to override the behavior configured in the schedule spec.
+
+`velero schedule unpause schedule-1 --unpause-triggers=false` will unpause the schedule but not trigger the backup immediately.
+
+`velero schedule unpause schedule-1 --unpause-triggers=true` will unpause the schedule and trigger the backup immediately.
+
+`velero schedule unpause schedule-1` will unpause the schedule and trigger the backup immediately. This is the same behavior as before.
```

Comment:
Shouldn't not passing the flag leave `Spec.UnpauseTriggers` as-is? i.e. if it's set to the default (true) value, then the backup is triggered immediately. however, if it's set to false (either because the last unpause set it to false, or the user set it to false explicitly when creating the schedule, shouldn't this previous value setting be honored?

Reply:
ok that make sense. I will clear the design up. Thanks!